templates/directorcontrol/globalsearchtagrestrictions.html

```diff
@@ -0,0 +1,38 @@
+$def with (tags)
+$:{RENDER("common/stage_title.html", ["Edit Global Community Tagging Restrictions", "Director Control Panel"])}
+
+<div class="content form clear">
+  <form method="POST" class="form skinny clear" action="/directorcontrol/globalsearchtagrestrictions">
+    <h3>Current Global Community Tagging Restrictions</h3>
+    <table>
+      <thead>
+        <tr>
+          <th width="50%">Added By</th>
+          <th width="50%">Tag</th>
+        </tr>
+      </thead>
+      <tbody>
+        $for entry in tags:
+          <tr>
+            <td style="text-align:center">$entry.login_name</td>
+            <td style="text-align:center">$entry.title</td>
+          </tr>
+      </tbody>
+    </table>
+    $:{CSRF()}
+    <h3>Global Community Tagging Restrictions Information</h3>
+    <p>
+      Tags added here will not be able to be added to the list of search tags for an item,
+      except by the owner of the item in question, or Weasyl staff. In addition to blocking individual tags, you can
+      forbid tag patterns by using <code>*</code> as a wildcard. Wildcard entries must have at least 
```

Comment:
How about:

Tag patterns can only use one <code>*</code> and must contain at least two additional characters.